x-pack/plugins/index_management/public/application/sections/home/data_stream_list/data_stream_table/data_stream_table.tsx

```diff
@@ -44,14 +45,27 @@ export const DataStreamTable: React.FunctionComponent<Props> = ({
     }),
     truncateText: true,
     sortable: true,
-    render: (name: DataStream['name']) => {
+    render: (name: DataStream['name'], dataStream: DataStream) => {
       return (
-        <EuiLink
-          data-test-subj="nameLink"
-          {...reactRouterNavigate(history, getDataStreamDetailsLink(name))}
-        >
-          {name}
-        </EuiLink>
+        <Fragment>
+          <EuiLink
+            data-test-subj="nameLink"
+            {...reactRouterNavigate(history, getDataStreamDetailsLink(name))}
+          >
+            {name}
+          </EuiLink>
+          {isManagedByIngestManager(dataStream) ? (
+            <Fragment>
+              &nbsp;
+              <EuiBadge color="hollow">
```

Comment:
What do you think about adding a tooltip or title to the badge to explain that "managed" means managed by Ingest Manager, specifically? Or maybe add this to the switch?